app/(components)/NoTicketExistsCard.jsx

```diff
@@ -0,0 +1,20 @@
+import Link from "next/link";
+
+const NoTicketExistsCard = () => {
+    return (
+        <div className="lg:grid grid-cols-2 xl:grid-cols-4 gap-4">
+            <div
+                className="flex flex-col hover:bg-card-hover bg-card rounded-md shadow-lg p-3 m-2"
+            >
+                <h4 className="mb-1">No Tickets Found</h4>
+                <hr className="h-px border-0 bg-page mb-2 " />
+                <p className="whitespace-pre-wrap">There are no tickets available at the moment.</p>
+
+                <div className="flex-grow"></div>
```

Comment:
is this intentional?

Reply:
No actually, I copied the card code from ticket card where it was there. I tried removing it and it works completely fine. May i remove it and commit?